chart/openfaas/templates/prometheus-dep.yaml

```diff
@@ -16,8 +16,10 @@ spec:
       labels:
         app: prometheus
       annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
How come?

Reply:
The K8s SD returns pod IPs and those don't work inside the mesh

Reply:
There is a way to make it work but it needs the Istio cert, here is an example https://github.com/stefanprodan/flagger/blob/master/artifacts/gke/istio-prometheus.yaml#L330